notebooks/111-detection-quantization/111-detection-quantization.ipynb

```diff
@@ -0,0 +1,773 @@
+    "                 (with values between 0 and 1) and metadata a dictionary: {\"filename\": path_to_image}\n",
+    "        \"\"\"\n",
+    "        image_path = self.images[index]\n",
+    "        image = cv2.imread(str(image_path))\n",
+    "        image = cv2.resize(image, self.target_size)\n",
+    "        image_id = self.image_ids[Path(image_path).name]\n",
+    "\n",
+    "        # image_info contains height and width of the annotated image\n",
+    "        image_info = [\n",
+    "            image for image in self.annotations[\"images\"] if image[\"id\"] == image_id\n",
+    "        ][0]\n",
+    "        # image_annotations contains the boxes and labels for the image\n",
+    "        image_annotations = [\n",
+    "            item\n",
+    "            for item in self.annotations[\"annotations\"]\n",
+    "            if item[\"image_id\"] == image_id\n",
+    "        ]\n",
+    "\n",
+    "        # annotations are in xmin, ymin, width, height format. Convert to\n",
+    "        # xmin, ymin, xmax, ymax and normalize to image width and height as\n",
+    "        # stored in the annotation\n",
+    "        target_annotations = []\n",
+    "        for annotation in image_annotations:\n",
+    "            xmin, ymin, width, height = annotation[\"bbox\"]\n",
+    "            xmax = xmin + width\n",
+    "            ymax = ymin + height\n",
+    "            xmin /= image_info[\"width\"]\n",
+    "            ymin /= image_info[\"height\"]\n",
+    "            xmax /= image_info[\"width\"]\n",
+    "            ymax /= image_info[\"height\"]\n",
+    "            target_annotation = {}\n",
+    "            target_annotation[\"category_id\"] = annotation[\"category_id\"]\n",
+    "            target_annotation[\"image_width\"] = image_info[\"width\"]\n",
+    "            target_annotation[\"image_height\"] = image_info[\"height\"]\n",
+    "            target_annotation[\"bbox\"] = [xmin, ymin, xmax, ymax]\n",
+    "            target_annotations.append(target_annotation)\n",
+    "\n",
+    "        item_annotation = (index, target_annotations)\n",
+    "        input_image = np.expand_dims(image.transpose(2, 0, 1), axis=0).astype(\n",
+    "            np.float32\n",
+    "        )\n",
+    "        return (\n",
+    "            item_annotation,\n",
+    "            input_image,\n",
+    "            {\"filename\": str(image_path), \"shape\": image.shape},\n",
+    "        )\n",
+    "\n",
+    "    def __len__(self):\n",
+    "        return len(self.images)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8cb85bf1-db51-4d5e-9238-4c24c3e13a8a",
+   "metadata": {},
+   "source": [
+    "#### Metric\n",
+    "\n",
+    "Define a metric to determine the performance of the model. For the Default Quantization algorithm that is used in this notebook, defining a metric is optional, but it can be used to compare the metric of the quantized INT8 model with the original FP IR model.\n",
```

Comment:
```suggestion
    "Define a metric to determine model's performance. For the Default Quantization algorithm used in this notebook, defining a metric is optional, but it can be used to compare the quantized INT8 model with the original FP IR model.\n",
```